repositories.gradle

```diff
@@ -8,9 +8,11 @@ repositories {
         }
     }
     maven {
-        url = "https://jitpack.io"
+        name "GTNH Maven"
```

Comment:
i wouldnt replace jitpack just add us maven on top

Reply:
Why though? All dependencies are on our maven already

Reply:
yes but if mavent offline we can fall back to jitpack